ansible/roles/local_storage/tasks/main.yml

```diff
@@ -21,3 +21,13 @@
   until: lv_result.failed == false
   retries: 60
   delay: 2
+
+- name: Create the localvolume for the Kafka brokers
+  k8s: 
+    state: present
+    definition: "{{ lookup('template', 'kafka-lv-brokers.yml.j2)' }}"
+
+- name: Create the localvolume for the Kafka zookeepers
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'kafka-lv-zookeepers.yaml.j2)' }}"
```

Comment:
If we require multiple localvolume objects we will can find a common template that we iterate over.  I'm thinking the vars would look something like:
```
local_volume_objects:
  - key: localstorage
    value: "true"
    storageclassname: local-sc
    volumes:
    - /dev/vg_ls/lv_tv00
    - /dev/vg_ls/lv_tv01
    - /dev/vg_ls/lv_tv02
    - /dev/vg_ls/lv_tv03
    - /dev/vg_ls/lv_tv04
    - /dev/vg_ls/lv_tv05
  - key: kafka
    value: "true"
    storageclassname: kafka-local-brokers
    volumes:
    - /dev/nvme0n1
  - key: kafka
    value: "true"
    storageclassname: kafka-local-zookeepers
    volumes:
    - /dev/nvme0n2
```

We can then have a single task that templates per item.  Thoughts?

Reply:
I indeed forgot about the LV creation in the kafka role, removed that for now. I agree that we can have a single task for the local volumes, imho it's not gaining readability that way, but that's just me. I'll change it according to your proposal and commit it.